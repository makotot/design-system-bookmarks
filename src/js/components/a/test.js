import React from 'react'
import A from './'
import renderer from 'react-test-renderer'

describe('A', () => {
  it('renders collectly', () => {
    const tree = renderer.create(<A href="/" />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
