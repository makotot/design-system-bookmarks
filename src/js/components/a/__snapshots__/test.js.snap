// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A renders collectly 1`] = `
<a
  className="c-anchor"
  href="/"
/>
`;
